gist/gist.py

```diff
@@ -138,19 +138,6 @@ def __init__(self, token, editor=None):
         """
         self.token = token
         self.editor = editor
-        self.proxies = {}
```

Comment:
With regards to the unit tests - this is actually core urllib stuff, not even requests. The unit test for that behavior is here:
https://github.com/python/cpython/blob/2.7/Lib/test/test_urllib.py#L147